facturacion_mexico/tests/test_fiscal_state_payment_entry.py

```
@@ -0,0 +1,276 @@
+import frappe
+from frappe.tests.utils import FrappeTestCase
+
+from facturacion_mexico.fiscal_state.payment_entry_state import (
+	_compute_actions,
+	_compute_facts,
+	_compute_messages,
+	get_payment_entry_fiscal_state,
+)
+
+
+def _mock_pe(
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_mock_pe`

(ANN202)